apiservers/engine/CLI-COMPATIBILITY.md

```diff
@@ -0,0 +1,274 @@
+# Docker CLI Compatibility
+VIC attempts to be compatible with Docker client (CLI); however, there are some differences.  The CLI differences are documented below, alphabetically.  Note, the operations listed as supported but differences does not mean incompatible.  It means most of these operations support either a smaller subset of the parameters or the request/response parameters have different meanings that apply to running in an ESX or vCenter environment.
```

Comment:
VIC attempts to be compatible with the Docker client (CLI); however, there are some differences.  The CLI differences are documented below, alphabetically.  Note, the operations listed as supported but with differences does not mean incompatible.  It means most of these operations support either a smaller subset of the parameters or the request/response parameters have different meanings that apply to running in an ESX or vCenter environment.